spark_auto_mapper_fhir/resources/schedule.py

```diff
@@ -0,0 +1,58 @@
+from typing import Optional, Union
+
+from pyspark.sql.types import StructType
+from spark_fhir_schemas.r4.resources.schedule import ScheduleSchema
+
+from spark_auto_mapper_fhir.complex_types.identifier import Identifier
+from spark_auto_mapper_fhir.complex_types.meta import Meta
+from spark_auto_mapper_fhir.complex_types.reference import Reference
+from spark_auto_mapper_fhir.extensions.extension_base import ExtensionBase
+from spark_auto_mapper_fhir.fhir_types.boolean import FhirBoolean
+from spark_auto_mapper_fhir.fhir_types.id import FhirId
+from spark_auto_mapper_fhir.fhir_types.list import FhirList
+from spark_auto_mapper_fhir.resources.device import Device
+from spark_auto_mapper_fhir.resources.fhir_resource_base import FhirResourceBase
+from spark_auto_mapper_fhir.resources.healthcare_service import HealthcareService
+from spark_auto_mapper_fhir.resources.location import Location
+from spark_auto_mapper_fhir.resources.patient import Patient
+from spark_auto_mapper_fhir.resources.practitioner import Practitioner
+from spark_auto_mapper_fhir.resources.practitioner_role import PractitionerRole
+from spark_auto_mapper_fhir.resources.related_person import RelatedPerson
+
+
+class Schedule(FhirResourceBase):
+    # noinspection PyPep8Naming
+    def __init__(
+        self,
+        id_: FhirId,
```

Comment:
I believe this field is optional.  Did you use the New->FhirResource template?  That would set these automatically.